.gitlab-ci.yml

```diff
@@ -62,24 +62,47 @@ test:perl5.30-mysql:
 # Triggers for dependent builds
 #
 
-# FIXME:
-#  - only partly tested so far
-#  - doesn't support PRs before they are merged
-#    (would need extended run condition and better selection of downstream branches)
+# The template. It doesn't presently support PRs before they are
+# merged (would need extended run condition and better selection of
+# downstream branches) - but then again, we do not trigger dependent
+# builds for PRs on Travis either.
 .dependent_template:
   stage: posttest
   # We want this to run even if any test jobs fail
   when: always
   only:
     - master
     - /^release/\d+$/
+
+# Actual trigger jobs
+
+# ensembl-rest; disabled for now because that repo a) hasn't got
+# GitLab-CI config yet, and b) is still on the list in
+# trigger-dependent-build.sh.
+.post:trigger_rest:
+  extends: .dependent_template
   trigger:
-    project: $DEPENDENT_PROJECT
+    project: ensembl-gh-mirror/ensembl-rest
     # Use the same branch as in this project
-    branch: $CI_COMMIT_REF_NAME
+    branch: ${CI_COMMIT_REF_NAME}
 
-# Example trigger job using the above, disabled for obvious reasons
-.post:trigger_foo:
+# Dependent builds on Travis
+# Relies on the secret variable TRAVIS_AUTH_TOKEN to actually work,
+# moreover the account associated with the token must have write
+# access to *all* dependent repositories; to be exact what it needs
+# is the Travis 'create_request' permission but Travis permissions are
+# generated from GitHub ones and it seems that in order to have
+# 'create_request' on the latter one requires 'write' on the former.
+post:trigger_travis:
```

Comment:
does this need to be `.post`?

Reply:
Good point, with the "when: always" condition in place all delegating it to the second stage did was delay the downstream build.